src/store/metrics.js

```diff
@@ -0,0 +1,224 @@
+import Vue from 'vue'
+import store from '@/store/index'
+import { MetricHelper } from '@/metric'
+
+export default {
+  namespaced: true,
+  state: {
+    metrics: {}
+  },
+  getters: {
+    getMetricKey: (state) => (metricName) => {
+      const metricEntry = Object.entries(state.metrics).find(([_, metric]) => metric.name === metricName)
+      return metricEntry === undefined ? undefined : metricEntry[0]
+    },
+    getMetricDrawState: (state) => (metricName) => {
+      const metric = Object.entries(state.metrics).find(([_, metric]) => metric.name === metricName)[1]
+      return {
+        drawMin: metric.drawMin,
+        drawAvg: metric.drawAvg,
+        drawMax: metric.drawMax
+      }
+    },
+    getAllMetricKeys: (state) => () => {
+      return Object.keys(state.metrics)
+    },
+    getAllMetrics: (state) => () => {
+      return Object.values(state.metrics)
+    }
+  },
```

Comment:
Why is `metrics.metrics` accessed without a getter so often? That seems inconsistent.